statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -4231,7 +4231,8 @@ def apply(self, endog, exog=None, refit=False, fit_kwargs=None,
         return res
 
     def plot_diagnostics(self, variable=0, lags=10, fig=None, figsize=None,
-                         truncate_endog_names=24):
+                         truncate_endog_names=24, auto_ylims=False,
+                         bartlett_confint=False, acf_kwargs=dict()):
```

Comment:
I think in this case we should still use `acf_kwargs=None` (even though you're not modifying it in the function) and then do an:

```
if acf_kwargs is None:
    acf_kwargs = {}
```

This is awkward (and not strictly necessary given your implementation), but I think it's good practice (e.g. in case someone later does something with this dict in the function without realizing that it will [modify the defaults in later calls](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python)).

Reply:
Done!